_posts/2022-01-28-data-prepper-1.2.1-performance-testing.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  "Data Prepper 1.2.1 Performance Testing"
+authors:
+  - sbayer
+date: 2022-01-28
+categories:
+  - technical-post
+
+excerpt: "Following the launch of log pipeline in Data Prepper 1.2, the Data Prepper Team is excited to share the results of our performance testing suite. Our goal is to create a tool that can simulate a set of real-world scenarios in any environment while maintaining compatibility with popular log ingestion applications."
+
+---
+
+Following the launch of log pipeline in [Data Prepper 1.2](https://www.opensearch.org/blog/technical-post/2021/12/Introducing-Data-Prepper-1.2.0-with-Log-Pipelines/), we are excited to share the results of our performance testing suite. Our goal is to create a tool that can simulate a set of real-world scenarios in any environment while maintaining compatibility with popular log ingestion applications. In all performance test results discussed below, the test environments and configurations are identical, except where the same option is not available for all applications. See testing limitations section for additional details.
+
+# TL;DR Data Prepper 1.2.1 vs Logstash 7.13.2 Results
```

Comment:
I didn't catch this the first time but did you explain why Logstash 7.13.x was used not the current version?